app/services/usps_in_person_proofing/proofer.rb

```diff
@@ -5,23 +5,34 @@ class Proofer
     AUTH_TOKEN_CACHE_KEY = :usps_ippaas_api_auth_token
     # Automatically refresh our auth token if it is within this many minutes of expiring
     AUTH_TOKEN_PREEMPTIVE_EXPIRY_MINUTES = 1.minute.freeze
+    # Sponsor ID to use for enhanced in-person proofing
+    USPS_EIPP_SPONSOR_ID = 36
+    # Assurance Level to use for enhanced in-person proofing
+    USPS_EIPP_ASSURANCE_LEVEL = '2.0'
 
     # Makes HTTP request to get nearby in-person proofing facilities
-    # Requires address, city, state and zip code.
+    # Requires address, city, state, zip code and vector of trust.
     # The PostOffice objects have a subset of the fields
     # returned by the API.
     # @param location [Object]
+    # @param vector_of_trust [Array<String>]
     # @return [Array<PostOffice>] Facility locations
-    def request_facilities(location)
+    def request_facilities(location:, vector_of_trust: nil)
```

Comment:
1. we're not actually using the contents of the `vector_of_trust` here, I think this could be easily if we added a boolean `enhanced_in_person_proofing: false` or something, so the calling controller can determine what to send?
2. This is a switch to keyword args, but all the other callsites are still using positional args, including the mock, that is a recipe for a bad time.

I feel pretty strongly about cleaning up (1) and don't particularly care if we we positional or keywords for (2) as long as it's consistent

Reply:
Agreed on both points.  See [dceae53](https://github.com/18F/identity-idp/pull/10750/commits/dceae53cdfc662f3daa64229a6cfef83ce5231f1).

Reply:
I personally feel a little more strongly on the point of positional vs. keyword arguments, in favor of using keyword arguments with booleans. I think the risk is that a future developer seeing a callsite like this for the first time:

```rb
locations = proofer.request_facilities(candidate, false)
```

It is very unclear what "false" refers to.